mcp-server/coneshare_mcp/client.py

```
@@ -0,0 +1,260 @@
+
+                try:
+                    error_detail = response.json()
+                except Exception:
+                    error_detail = response.text
+
+                return {
+                    "error": True,
+                    "status": response.status_code,
+                    "detail": error_detail,
+                }
+            except Exception as exc:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)